Makefile

```diff
@@ -102,13 +108,19 @@ manifests/operators: operator-source
 manifests: manifests/00-catalog.yaml manifests/operators
 
 .PHONY: operator-source
-operator-source: 
-	$(call checkout_operator,openshift,dedicated-admin-operator)
+operator-source:
+	for operator in $(OPERATORS); do \
+		org="$$(echo $$operator | cut -d / -f 1)" ; \
+		reponame="$$(echo $$operator | cut -d / -f 2-)" ; \
+		echo "org = $$org reponame = $$reponame" ; \
+		$(call checkout_operator,$$org,$$reponame) ;\
+		echo ;\
+	done
```

Comment:
Why have a loop over variable instead of just calling out to the function per repo as it was?  I wouldn't recommend overriding it with CLI vars.  It's just more complicated this way.

Reply:
@jewzaam I'm imagining we'll have more than one operator and it's easier to append to the list than duplicate a `$(call ...)` command.

Reply:
You're editing the Makefile either way.  Felt just another call statement would be very straight forward.  1 line of code per operator, no loop, no var.  Simple.

Reply:
so in my opinion, Makefiles should be pretty static once you get things going. There's lots of dynamic bits that you can do. Iterating over a loop is preferable, I think. It's possible to do things as well as extract the list of operators into a plain text file, and then read that in and iterate over it.

Reply:
The list of operators will be pretty static once we have them developed and integrated.  I just feel that putting the list into a file that we then have to load an iterate over adds unnecessary complexity in the makefile and adds additional unnecessary places for configuration to be setup.  It's explicit if it's in the target, isn't a var, and isn't in a file.

Reply:
I've slept on this and coming back to it I think my main concern about unrolling the loop is the way in which it's unrolled. I believe there's more chance for typos given the Makefile syntax of `$(call checkout_operator,openshift,pagerduty-operator)` compared to appending to a list, `OPERATORS := openshift/dedicated-admin-operator openshift/pagerduty-operator`. We don't often use Makefiles and coming back to remember how to add a new operator to the list is something I want to make as easy and reviewable as possible. For example, imagine you're adding a new operator six months from now and compare:

```diff
diff --git a/Makefile b/Makefile
index 06d1b5d..c9c8067 100644
--- a/Makefile
+++ b/Makefile
@@ -50,7 +50,7 @@ SOURCE_DIR := operators
 
 # List of github.org repositories containing operators
 # This is in the format username/reponame
-OPERATORS := openshift/dedicated-admin-operator
+OPERATORS := openshift/dedicated-admin-operator openshift/pagerduty-operator
 
 .PHONY: default
 default: build
```

Compare to:

```diff
--- Makefile.orig	2019-04-10 08:30:35.000000000 -0400
+++ Makefile	2019-04-10 08:35:56.000000000 -0400
@@ -112,6 +112,7 @@
 .PHONY: operator-source
 operator-source:
 	$(call checkout_operator,openshift,dedicated-admin-operator)
+	$(call checkout_operator,openshift,pagerduty-operator)
 
 .PHONY: catalog
 catalog: operator-source
```

You see a lot of `; \` in the Makefile in other targets, do you need to copy that here? As a reviewer I'm going to wonder the same thing. People reviewing the changes must also be aware that there can't be any whitespace around the `,`, so the natural inclination to add whitespace for readability (like Go demands and Python style encourages) will break it.

I believe the loop (in this PR, for this target) is stable and the path of least resistance is the loop style.